ntbs-service/Startup.cs

```diff
@@ -515,16 +514,16 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             {
                 app.UseForwardedHeaders();
                 app.UseDeveloperExceptionPage();
-                app.UseWebpackDevMiddleware(new WebpackDevMiddlewareOptions
-                {
-                    HotModuleReplacement = true, ConfigFile = "webpack.dev.js"
-                });
+
+                // Hot module reloading has been made more difficult in .NET 5 because UseWebpackDevMiddleware was removed
+                // See https://github.com/dotnet/AspNetCore/issues/12890 for discussion of this change
```

Comment:
I'm not sure this comment makes sense - you're commenting on something that's not there. I think the place that we may want to comment on this is in the readme - it may make sense there to say "we used to support this, but unfortunately we can't anymore"

Reply:
Yeah I agree, I've removed the now-false promises of hot module reloading from the README and explained why we don't support this any more as you suggested 🙂